server/tests/events.test.js

```diff
@@ -0,0 +1,162 @@
+const mongoose = require("mongoose")
+const createServer = require("./testingTools")
+var alumniSchema = require('../models/alumniSchema')
+var userSchema = require('../models/userSchema')
+var studentSchema = require('../models/studentSchema')
+var schoolSchema = require('../models/schoolSchema')
+var majorSchema = require('../models/majorSchema')
+var eventSchema = require('../models/eventSchema')
+const request = require('supertest')
+var COUNTRIES = require("../countries").COUNTRIES
+
+const app = createServer()
+
+beforeAll(async () => {
+    process.env.NODE_ENV = 'test'
+    mongoose.connect("mongodb://localhost:27017/test-test", { useNewUrlParser: true })
+    await request(app).get('/util/seed/')
+})
+
+afterAll(async () => {
+    // await request(app).get('/data/clear/all')
+    const collections = Object.keys(mongoose.connection.collections)
+    for (const collectionName of collections) {
+        const collection = mongoose.connection.collections[collectionName]
+        await collection.deleteMany()
+    }
+    await mongoose.connection.close()
+    delete process.env.NODE_ENV
+})
+
+test('students not allowed to create new events', async () => {
+    var student = await studentSchema.findOne()
+    var userID = student.user
+    const res = await request(app).post(`/events/create/${userID}`)
+        .send({
+            eventName: 'how dinosaurs helped establish modern architecture'
```

Comment:
how do I RSVP to this

Reply:
In the MVP version RSVP'ing won't be a part of the functionality. There is a follow up story to address this though. Mayhaps this is something we can talk about during stand up as well.